e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -741,7 +741,6 @@ describe("#39152 sharing an unsaved question", () => {
             });
           });
         });
```

Comment:
```suggestion
        });

```

there should be an empty line, otherwise lint will fail on master

Reply:
Thank you! 🙏